internal/transport/test/receive_pack.go

```diff
@@ -199,27 +228,28 @@ func (s *ReceivePackSuite) TestSendPackOnNonEmptyWithReportStatusWithError() {
 	endpoint := s.Endpoint
 	full := false
 	fixture := fixtures.Basic().ByTag("packfile").One()
-	req := packp.NewReferenceUpdateRequest()
+	req := &transport.PushRequest{}
 	req.Commands = []*packp.Command{
 		{Name: "refs/heads/master", Old: plumbing.ZeroHash, New: plumbing.NewHash(fixture.Head)},
 	}
-	req.Capabilities.Set(capability.ReportStatus)
+	// req.Capabilities.Set(capability.ReportStatus)
 
-	report, err := s.receivePackNoCheck(endpoint, req, fixture, full)
+	// report, err := s.receivePackNoCheck(endpoint, req, fixture, full)
+	err := s.receivePackNoCheck(endpoint, req, fixture, full)
 	// XXX: Recent git versions return "failed to update ref", while older
 	//     (>=1.9) return "failed to lock".
 	s.Regexp(regexp.MustCompile(".*(failed to update ref|failed to lock).*"), err)
-	s.Equal("ok", report.UnpackStatus)
-	s.Len(report.CommandStatuses, 1)
-	s.Equal(plumbing.ReferenceName("refs/heads/master"), report.CommandStatuses[0].ReferenceName)
-	s.Regexp(regexp.MustCompile("(failed to update ref|failed to lock)"), report.CommandStatuses[0].Status)
+	// s.Equal("ok", report.UnpackStatus)
+	// s.Len(report.CommandStatuses, 1)
+	// s.Equal(plumbing.ReferenceName("refs/heads/master"), report.CommandStatuses[0].ReferenceName)
+	// s.Regexp(regexp.MustCompile("(failed to update ref|failed to lock)"), report.CommandStatuses[0].Status)
```

Comment:
If we are commenting this code in between PRs, it would be good to add a TODO justifying it.